app/admin/equipment/gateways.rb

```diff
@@ -439,6 +439,10 @@ def resource_params
           f.input :diversion_domain
           f.input :diversion_rewrite_rule
           f.input :diversion_rewrite_result
+
+          f.input :pai_send_mode_id, as: :select, include_blank: false,
+                                     collection: Gateway.pai_send_modes
```

Comment:
after constant `Gateway::PAI_SEND_MODES` added 

replace the line with `collection: Gateway::PAI_SEND_MODES.invert`